src/test/common/page/guidance.js

```diff
@@ -4,14 +4,23 @@ const Page = require('./page')
 
 const { PAGES, getPageMetadata } = require('../../../web/routes/guidance')
 
+const APPLY_PAGE_NAME = 'Apply for Healthy Start'
+const START_BUTTON_CLASS = 'govuk-button--start'
+
 /**
  * Page object for all the guidance pages. This may extend Page, but as it represent multiple static page, it must
  * implement its own methods for navigation, but can use all the find methods from Page when required.
  */
 class Guidance extends Page {
+  constructor (driver) {
+    super(driver)
+    this.sessionId = null
+  }
+
   async openGuidancePage (baseURL, pageName, lang = 'en') {
     try {
       await this.openPage(`${baseURL}${this.getPageWithTitle(PAGES, pageName).path}`, lang)
+      this.sessionId = await this.getCurrentSessionId()
```

Comment:
Guidance pages to not use base page `open` method so session ID needs to be stored at this point